storage/zfs/README.md

```diff
@@ -122,3 +122,22 @@ talosctl mounts | grep <mount of zfs raid you specified (/var/...)>
 ```
 
 You should see the mount.
+
+## Use existing pools
+
+If you have existing pools, they will be automatically detected and used.
+
+However, you may need to enter a debug shell and modify the pool configuration in order for them to work with talos.
+Turn off samba and nfs share settings.
+In addition, some SELinux pool policies are currently incompatible with talos. If you see errors like this:
+```bash
+node_name: kern: warning: [2025-09-03T00:34:28.924235026Z]: SELinux: Unable to set superblock options before the security server is initialized
```

Comment:
Did you explicitly disable SELinux (e.g. by passing `selinux=0`)? What version of Talos did you find this issue?

By default, Talos loads a policy in permissive mode. `selinux=0` disables the SELinux LSM, and `enforcing=1` makes the policy enforced (not supported nor tested with extensions enabled).

This seems to be a somewhat rare warning, does this prevent ZFS volumes from being mounted?

Reply:
If it fails without selinux=0, then I guess it could potentially be caused by missing labels. Talos does not have SELinux labels coming from a Rocky install, so you'll have to disable those via volume options anyway I guess.

I wonder if a volume that has SELinux on, but no foreign labels mounts fine.

But so far this step seems to make sense